src/service/repo/signedMessageRepo.ts

```diff
@@ -3,6 +3,7 @@ import api from '../config';
 import {ChannelData} from '../../../types/repo/ChannelData';
 import {
   ChannelTypeEnum,
+  EditChannelPostTyoe,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the import statement. It should be `EditChannelPostType` instead of `EditChannelPostTyoe`.
```diff
-  EditChannelPostTyoe,
+  EditChannelPostType,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->